### PR TITLE
Add stale and auto-comment workflows

### PR DESCRIPTION
**Pull request by awharn**

### CODE REVIEW

**Review by t1m0thyj — APPROVED** [review]
This will be awesome for managing new issues and PRs 🎉

File: .github/label-actions.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+bug:
+  comment: |
+    Thank you for creating a bug report. If you haven't already, please ensure you have provided steps to reproduce it and as much context as possible.
+enhancement:
+  comment: |
+    Thank you for raising this issue. 
+    The community has 90 days to upvote 👍 the issue.
+    If it receives 10 upvotes, we will move it to our backlog. If not, we will close it.
```
Comment (zFernand0) [inline .github/label-actions.yml:8]: I think it's going to be a little hard to reach 10 upvotes.
Should we lower it down to 5 ?

**Review by zFernand0 — APPROVED** [review]

**Review by ATorrise — APPROVED** [review]

**Review by gejohnston — APPROVED** [review]
This should lessen our administrative burden.

**Review by anaxceron — APPROVED** [review]
Timelines no longer conflict, and the auto responses look good to me.

**Pull request merged**